common/lib/container-definitions/package.json

```diff
@@ -491,6 +491,27 @@
       },
       "0.46.0": {
         "InterfaceDeclaration_IContainerContext": {
+          "forwardCompat": false,
+          "backCompat": false
+        },
+        "InterfaceDeclaration_ICodeDetailsLoader": {
+          "backCompat": false
+        },
+        "InterfaceDeclaration_ICodeLoader": {
+          "backCompat": false
+        },
+        "InterfaceDeclaration_IFluidModule": {
+          "backCompat": false
+        },
+        "InterfaceDeclaration_IFluidModuleWithDetails": {
+          "backCompat": false
+        },
+        "InterfaceDeclaration_IProvideRuntimeFactory": {
+          "forwardCompat": false,
+          "backCompat": false
+        },
+        "InterfaceDeclaration_IRuntimeFactory": {
+          "backCompat": false,
```

Comment:
Why would CodeDetailsLoader changes be here ?

Reply:
There were few build failures in container-definitions

Reply:
Next time, please separate these changes in their own PRs. If they are causing build failures, that makes it even more important to move smaller changes/fixes through as a separate PR.